raphtory/src/core/tgraph.rs

```diff
@@ -312,6 +289,23 @@ impl TemporalGraph {
         self.layers[layer].add_edge_with_props(t, src_pid, dst_pid, props)
     }
 
+    pub(crate) fn delete_edge<T: InputVertex>(&mut self, t: i64, src: T, dst: T, layer: usize) {
+        let src_id = src.id();
+        let dst_id = dst.id();
+        // mark the times of the vertices at t
+        self.add_vertex(t, src)
```

Comment:
Is the vertices getting some addition in their history the semantics we want for deleted edges?

Reply:
Need to make a ticket for this

Reply:
Currently required to make sure the ids are properly assigned